packages/cli/src/commands/plugin/rollup.config.ts

```diff
@@ -36,7 +44,15 @@ export default {
     }),
     commonjs({
       include: ['node_modules/**', '../../node_modules/**'],
-      exclude: ['**/*.stories.js'],
+      exclude: ['**/*.stories.*', '**/*.test.*'],
+      namedExports: Object.keys(NAMED_EXPORTS_OVERRIDES).reduce(
```

Comment:
Should be fine to remove the namedExports workaround if you rebase on master. Keep the exclude fix tho :grin: